src/components/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { FilterType } from '../App';
+
+type Props = {
+  todos: Todo[];
+  selectedFilter: FilterType;
+  handleDeleteAllCompleted: () => Promise<void>;
+  showFilteredTodos: (filterType: FilterType) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  selectedFilter,
+  handleDeleteAllCompleted,
+  showFilteredTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.filter(todo => !todo.completed).length} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: selectedFilter === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => showFilteredTodos('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: selectedFilter === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => showFilteredTodos('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: selectedFilter === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => showFilteredTodos('completed')}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method